piccolo3/server/PiccoloCoolboxControl.py

```diff
@@ -26,11 +26,91 @@
 import asyncio
 from .PiccoloComponent import PiccoloBaseComponent, PiccoloNamedComponent, \
     piccoloGET, piccoloPUT, piccoloChanged
+import serial
 import logging
 
 from random import randint
 
 
+class PiccoloSerialConnection(PiccoloNamedComponent):
+    """Manage serial connection for controlling and managing the coolbox"""
+
+    def __init__(self, serial_port="/dev/ttyUSB0"):
+        self.verbose = False
```

Comment:
I wouldn't bother with the verbose flag - just use the logging tools. i tend to have these print statements in during the development. having said that, print doesn't work very well, so I tend to use logging

Reply:
Good point. I'll leave them in until everything is working as like you say they will be useful for debugging, but will remove them before merging coolbox with main